src/main/resources/static/domain-events-api.yml

```diff
@@ -672,6 +706,54 @@ components:
         - assessedAt
         - assessedBy
         - decision
+    PlacementApplicationWithdrawnEnvelope:
+      type: object
+      properties:
+        id:
+          $ref: '#/components/schemas/EventId'
+        timestamp:
+          type: string
+          example: '2022-11-30T14:53:44'
+          format: date-time
+        eventType:
+          type: string
+          example: approved-premises.placement-application.withdrawn
+        eventDetails:
+          $ref: '#/components/schemas/PlacementApplicationWithdrawn'
+      required:
+        - id
+        - timestamp
+        - eventType
+        - eventDetails
+    PlacementApplicationWithdrawn:
+      type: object
+      properties:
+        applicationId:
+          $ref: '#/components/schemas/ApplicationId'
+        applicationUrl:
+          $ref: '#/components/schemas/ApplicationUrl'
+        placementApplicationId:
+          $ref: '#/components/schemas/PlacementApplicationId'
+        personReference:
+          $ref: '#/components/schemas/PersonReference'
+        deliusEventNumber:
+          $ref: '#/components/schemas/DeliusEventNumber'
+        withdrawnAt:
+          type: string
+          example: '2022-11-30T14:51:30'
+          format: date-time
+        withdrawnBy:
+          $ref: '#/components/schemas/WithdrawnBy'
+        withdrawalReason:
+          type: string
```

Comment:
do you think it would be helpful to have an example here e.g.  `RELATED_APPLICATION_WITHDRAWN`?

Reply:
good idea, done